test/chainlinkAggregator.test.ts

```diff
@@ -0,0 +1,128 @@
+import { Signer } from '@ethersproject/abstract-signer'
+import chai from 'chai'
+import chaiAsPromised from 'chai-as-promised'
+import hre from 'hardhat'
+import { ChainlinkAggregator } from '../types/typechain'
+import { getMarket } from '../tasks'
+import { Address } from '../types/custom/config-types'
+import { ETH_ADDRESS } from '../consts'
+
+chai.should()
+chai.use(chaiAsPromised)
+
+const { deployments, ethers, contracts, getNamedSigner } = hre
+
+describe('Chainlink Aggregator', async () => {
+  let chainlinkAggregator: ChainlinkAggregator
+  let deployer: Signer
+  let sourceTokenAddress: Address
+  let destinationTokenAddress: Address
+  let aggregatorAddressToAdd: Address
+
+  // Setup for global tests
+  beforeEach(async () => {
+    deployer = await getNamedSigner('deployer')
+  })
+
+  describe('Add an additional chainlink aggregator', () => {
+    // Setup for snapshot tests
+    beforeEach(async () => {
+      // Get snapshot
+      await deployments.fixture('chainlink')
+      chainlinkAggregator = await contracts.get('ChainlinkAggregator', { from: deployer })
+
+      // Set up addresses
+      sourceTokenAddress = '0x7fc66500c84a76ad7e9c93437bfc5ac33e2ddae9' // AAVE
+      destinationTokenAddress = ETH_ADDRESS // ETH
+      aggregatorAddressToAdd = '0x6Df09E975c830ECae5bd4eD9d90f3A95a4f88012' // AAVE<>ETH
```

Comment:
these also won't change so they are unnecessarily being reset to the same values on every test

Reply:
got it, moved to initial setup